packages/components/alert-dialog/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@spark-ui/alert-dialog",
-  "version": "0.4.0",
+  "version": "1.0.0-0",
```

Comment:
Why `1.0.0-0` instead of `1.0.0`?

Reply:
If I set it to `1.0.0` it will publish `1.1.0` as a first version